atomix/src/test/java/com/atomix/exampletest/ClientExampleTest.java

```diff
@@ -0,0 +1,38 @@
+package com.atomix.exampletest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+import io.atomix.AtomixClient;
+import io.atomix.catalyst.transport.Address;
+import io.atomix.catalyst.transport.netty.NettyTransport;
+import io.atomix.collections.DistributedMap;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+public class ClientExampleTest {
+
+    @Test
+    public void ExampleTest() throws InterruptedException {
```

Comment:
Let's use the naming convention specified in the documentation

Reply:
Need to check-back the documentation what I am missing here.